lib/internal/abort_controller.js

```diff
@@ -29,7 +29,17 @@ const {
   }
 } = require('internal/errors');
 
+const {
+  validateUint32,
+} = require('internal/validators');
+
+const {
+  DOMException,
+} = internalBinding('messaging');
+
```

Comment:
```suggestion

const { setTimeout } = require('timers');

```
`setTimeout()` is used later on.